bot/cogs/fun.py

```diff
@@ -39,6 +39,14 @@ def __init__(self, bot: Bot):
         if "dabato" in message.content:
             await message.add_reaction("🤔")
 
+        # React FBI OPEN UP if message contains trigger words.
+        triggers = ["child", "fbi", "loli", "hentai", "illegal", "maltego"]
+        reactions = ["🇫", "🇧", "🇮", "⚫", "🇴", "🅿️", "🇪", "🇳", "⚪", "🇺", "🇵"]
+        for trigger in range(0, len(triggers) - 1):
+            if triggers[trigger] in message.content:
+                for reaction in range(0, len(reactions) - 1):
```

Comment:
Actually, not sure how I missed this but why are you iterating over a range and then using that as an index? Just iterate over the lists.

Reply:
In addition, instead of having an outer loop to iterate over trigger words, it can be replaced with `any()`:
`if any(trigger in message.content.lower() for trigger in triggers)`

`any()` has the advantage of immediately exiting as soon as one of the triggers is found, rather than continuing to find more triggers.